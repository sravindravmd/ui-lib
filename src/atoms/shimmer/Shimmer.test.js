import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { Shimmer } from './'

afterEach(cleanup)

describe('<Shimmer/>', () => {
  it('renders correctly', () => {
    const { container } = render(<Shimmer />)
    expect(container).toMatchSnapshot()
  })
})
