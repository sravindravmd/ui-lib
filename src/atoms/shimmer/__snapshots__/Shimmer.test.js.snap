// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Shimmer/> renders correctly 1`] = `
<div>
  <div
    class="sc-bdfBwQ aurdu"
  >
    <div
      class="shimmer-item"
    >
      <div
        class="sh-first shine"
      />
      <div
        class="sh-second shine"
      />
      <div
        class="sh-third shine"
      />
      <div
        class="sh-fourth"
      >
        <div
          class="sh-f-1"
        >
          <span
            class=" shine"
          />
        </div>
        <div
          class="sh-f-2"
        >
          <div
            class="sh-fc-1  shine"
          />
          <div
            class="sh-fc-2  shine"
          />
        </div>
      </div>
      <div
        class="sh-fourth"
      >
        <div
          class="sh-f-1"
        >
          <span
            class=" shine"
          />
        </div>
        <div
          class="sh-f-2"
        >
          <div
            class="sh-fc-1  shine"
          />
          <div
            class="sh-fc-2  shine"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
